docs/maui/TOC.yml

```diff
@@ -65,9 +65,11 @@ items:
   items:
     - name: Overview
       href: markup/markup.md
-    - name: "AbsoluteLayout extensions"
+    - name: "AbsoluteLayout Extensions"
```

Comment:
In regard to my other comment, how does that hold up for entries in the TOC?

Reply:
Looks like it should also be `AbsoluteLayout extensions`